internal/core/command/config.go

```diff
@@ -29,6 +29,7 @@ type ConfigurationStruct struct {
 	Registry    config.RegistryInfo
 	Service     config.ServiceInfo
 	SecretStore vault.SecretConfig
+	Startup     config.StartupInfo
```

Comment:
> the configuration.toml doesn't explicitly need values

You're technically correct.  However, there isn't documentation associated with the individual service configuration.toml files (and if there is, you haven't updated it to reflect these new potential entries).  The ask is to include entries in each configuration.toml file to make them explicit.  Otherwise, the only way to know that it's possible to add these to the configuration.toml file is to read the code.

Consider updating the StartupTimer implementation to eliminate the duplicate hard-coded values (which would address your multi-step change concern).

Reply:
One possibility is to add them to .toml but comment them out, so there's not required but are documented but their commented-out presence.  Tonight I'll just add as requested.

Accepted.